src/main/kotlin/org/jetbrains/research/testspark/display/TestCasePanel.kt

```diff
@@ -619,6 +655,13 @@ class TestCasePanel(
      */
     fun isRunEnabled() = runTestButton.isEnabled
 
+    /**
+     * Checks if the test case code has been modified before the applying to file.
+     *
+     * @return true if the final version of the code not differ from the initial one on the first request page.
+     */
+    fun isGlobalModified(): Boolean = testCase.testCode != initialCodes[0]
+
```

Comment:
The method name does not describe what it does. Maybe something like `isTestCaseModified`? Because it is unclear what `Global` means in this context.